AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py

```diff
@@ -0,0 +1,68 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+# Test Case Title : Check that the basic client connect test works
+
+
+# fmt: off
+class Tests():
```

Comment:
Are all these variables used? Might delete the ones we dont need, for example "Tests.found_unexpected_lines " is never used.